external-crates/move/crates/move-compiler/tests/linter/true_positive_excessive_nesting.move

```diff
@@ -0,0 +1,30 @@
+module 0x42::excessive_nesting_true_positive {
+
+    #[allow(unused_assignment)]
+    fun test_true_positive_direct_nesting() {
+        let value = 100;
+        
+        // Direct nesting that should be combined with &&
+        if (value > 50) {
```

Comment:
You don't need `{` in move for the body of an `if`, please add the appropriate testing